uaclient/api/u/apt_news/current_news/v1.py

```diff
@@ -0,0 +1,32 @@
+from typing import Optional
+
+from uaclient.api.api import APIEndpoint
+from uaclient.api.data_types import AdditionalInfo
+from uaclient.config import UAConfig
+from uaclient.data_types import DataObject, Field, StringDataValue
+from uaclient.files.state_files import apt_news_raw_file
+
+
+class CurrentNewsResult(DataObject, AdditionalInfo):
+    fields = [
+        Field("current_news", StringDataValue, required=False),
+    ]
+
+    def __init__(self, *, current_news: Optional[str]):
+        self.current_news = current_news
+
+
+def current_news() -> CurrentNewsResult:
+    return _current_news(UAConfig())
+
+
+def _current_news(cfg: UAConfig) -> CurrentNewsResult:
+    return CurrentNewsResult(current_news=apt_news_raw_file.read())
```

Comment:
Shouldn't the api try to fetch/update apt new when it is called ? My concern is that could be some conditions here we would output outdated messages to the user

Reply:
The message gets updated asynchronously every `apt update` at most once per day - and I don't think we want to change that cadence. The purpose of this API is for displaying the news in update-manager, similar to how it is displayed in `apt upgrade`, so I think it should always display the same message as `apt upgrade`.

I did consider adding a separate endpoint like "force_update_apt_news" to force a fetch/update outside of the normal `apt update` but since no-one is asking for it I figured it wasn't worth the effort (yet - until someone does ask for it :upside_down_face: )